CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+## 2.1.1 (Unreleased)
```

Comment:
Last ask before we merge. According to [semver](https://semver.org/):
```
    MAJOR version when you make incompatible API changes,
    MINOR version when you add functionality in a backwards-compatible manner, and
    PATCH version when you make backwards-compatible bug fixes.
```
Could you just set this as  `v2.2.0` for now as you have done more than just fixes. We will tag it `v2.2.0-alpha.1` so that you can use it for provider `go.mod` and bundle a few more PRs before we get to `v2.2.0` later on.

Thanks!
```suggestion
## v2.2.0 (Unreleased)
```

Reply:
Fixed.